src/lightning_app/components/serve/python_server.py

```diff
@@ -33,7 +28,9 @@ class Image(BaseModel):
 
     @staticmethod
     def _get_sample_data() -> Dict[Any, Any]:
-        imagepath = Path(__file__).absolute().parent / "catimage.png"
+        sep = "\\" if sys.platform == "win32" else "/"
```

Comment:
You can use `os.sep` here

Reply:
But this is failing on all platforms.. :(

Reply:
Done !